internals/overlord/servstate/state-diagram.dot

```diff
@@ -6,6 +6,7 @@ digraph service_state_machine {
     node [penwidth=3 shape=box fontsize=24] initial
     node [penwidth=1]
     initial -> starting [label="start"]
+    starting -> backoff [label="autorestart"]
```

Comment:
The label changed to `exited before okay wait elapses\n(action \"restart\")`.

Other state is still valid because if we call `stop()` when the service is still in the starting state, it will transfer to terminating, right? We added this feature to allow services to be stopped in the starting state.